conman/routes/managers.py

```diff
@@ -6,6 +6,8 @@
 
 class RouteManager(PolymorphicManager):
     """Helpful methods for working with Routes."""
+    silence_use_for_related_fields_deprecation = True
```

Comment:
Suggested in: https://docs.djangoproject.com/en/1.10/releases/1.10/#manager-use-for-related-fields-and-inheritance-changes